src/db.rs

```diff
@@ -407,15 +431,18 @@ mod tests {
             allowed_build_ids: vec![0u32, 1u32, 2u32],
         };
 
-        let tpid = insert_policy(&tinspol)?;
+        let tpid = aw!(insert_policy(&tinspol))?;
 
         let secid_uuid = Uuid::new_v4().as_hyphenated().to_string();
         let sec_uuid = Uuid::new_v4().as_hyphenated().to_string();
 
-        insert_secret(&secid_uuid, &sec_uuid, Some(tpid))?;
+        aw!(insert_secret(
+            &secid_uuid,
+            &sec_uuid,
+            Option::<u64>::Some(tpid)
+        ))?;
```

Comment:
I wonder why the explicit `Option::<u64>` is needed here. Is `tpid` no longer a `u64` ?